baseplate/sidecars/event_publisher.py

```diff
@@ -92,6 +93,9 @@ def __init__(self, metrics_client: metrics.Client, cfg: Any):
         self.key_name = cfg.key.name
         self.key_secret = cfg.key.secret
         self.session = requests.Session()
+        self.session.headers.update(
+            {"User-Agent": f"baseplate-{self.__class__.__name__}/{baseplate_version}"}
```

Comment:
Should this specify that it's `baseplate.py` since we also have `baseplate.go` now?